src/components/sdk/MeetingControls/docs/AudioInputVFControl.stories.mdx

```diff
@@ -0,0 +1,58 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import AudioInputVFControl from '../AudioInputVFControl';
+
+<Meta title="SDK Components/MeetingControls/AudioInputVFControl" />
+
+# AudioInputVFControl
+
+The `AudioInputVFControl` component renders a `ControlBarButton` with pop over menu options to select through multiple audio input options,
+and provides the option of voice focus on the bottom. This component should be used within `VoiceFocusProvider` and `meetingProvider`. 
+Make sure you have set `enableWebAudio` as `true` in the meeting config or the voice focus feature won't work.
+
+When you click the device button, the local audio input toggles between muted/unmuted state. When you click the voice focus button, 
+the voice focus feature toggles between on/off state. 
+
+If you want to apply the voice focus feature, try this `AudioInputVFControl`. If not, `AudioInputControl` is a better choice. 
```

Comment:
We provide it in example below, but should we call out the you need to add `enableWebAudio` as true to the `MeetingConfig` while using the `MeetingProvider`?

Reply:
yes, we have called it out in line 10